profiles/sssd/REQUIREMENTS

```diff
@@ -25,3 +25,6 @@ Make sure that SSSD service is configured and enabled. See SSSD documentation fo
 - with-gssapi is selected, make sure that GSSAPI authenticaiton is enabled in SSSD        {include if "with-gssapi"}
   - set pam_gssapi_services to a list of allowed services in /etc/sssd/sssd.conf          {include if "with-gssapi"}
   - see additional information in pam_sss_gss(8)                                          {include if "with-gssapi"}
+
+- with-homed is selected, make sure that the system-homed service is enabled              {include if "with-homed"}
```

Comment:
Should this have been updated to `with-systemd-homed` after the textual change to the feature name?